### ISSUE TITLE
Bounding box of volume is flat on Z

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/scenery#168]
```
Node.OrientedBoundingBox volBB = v.generateBoundingBox();

System.out.println( volBB.getMin() + " " + volBB.getMax() );
```        
yields
```
[[-1.0, -1.0, 0.0]] [[1.0, 1.0, 0.0]]
```
for a volume that clearly occupies all 3 dimensions

**Comment by skalarproduktraum** [comment]
That is indeed a bug and arises as Volume does not override generateBoundingBox() as it should. Sorry, I'll fix this asap.

**Comment by skalarproduktraum** [comment]
Fixed in #195.
